### PR TITLE
Latest sync for new release

### PR DESCRIPTION
**Pull request by phsong-google**
Updated change from internal source

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Baseline: [b5e34db]
 * Minor documentation and syntax fixes.
 
 ### Deprecations
-* Deprecated `GREYFail` in favor of `GREYFailWithDetails`.
+* Deprecated GREYFail in favor of GREYFailWithDetails.
```
Comment (sid-github) [inline CHANGELOG.md]: This change needs to be reverted.

File: gem/lib/earlgrey/version.rb [hunk 2]
```diff
@@ -14,5 +14,5 @@
 #  limitations under the License.
 
 module EarlGrey
-  VERSION = '0.1.1'.freeze unless defined? ::EarlGrey::VERSION
+  VERSION = '0.1.0'.freeze unless defined? ::EarlGrey::VERSION
```
Comment (sid-github) [inline gem/lib/earlgrey/version.rb]: revert this, gem is at 0.1.1

**Comment by sid-github** [comment]
LGTM

**Comment by tirodkar** [comment]
Fixes https://github.com/google/EarlGrey/issues/333
Fixes https://github.com/google/EarlGrey/issues/324

File: gem/README.md [hunk 3]
```diff
@@ -62,10 +62,8 @@ rake warn                  # Check for warnings
 
 The install command does the following by default:
 
-- Adds BridgingHeader.h and EarlGrey.swift to test target
```
Comment (bootstraponline) [inline gem/README.md]: we still add EarlGrey.swift to the test target

Comment (phsong-google) [inline gem/README.md]: Done

**Comment by tirodkar** [comment]
LGTM

**Pull request merged**